### DISCUSSION TITLE
A question on SoundAnalyzer.h

### DISCUSSION THREAD

**Discussion by atuline** [PlummersSoftwareLLC/NightDriverStrip#61] [Q&A]
In reviewing the min and max values you're calculating in the above file, I noted on line 118 at:

https://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/788c4cd1dae22cc5c6033c73dcd0991451d1c51c/include/soundanalyzer.h#L118

you're only using the first 4 bands. Am wondering why you don't use NUM_BANDS like you do on line 151?

**Comment by robertlipe** [comment]
For the intensely curious, this was resolved in https://github.com/PlummersSoftwareLLC/NightDriverStrip/commit/745d1399a86bf4d29129c9eaed1a943a54e3bf45

That entire method/file has been overhauled a few times since then.

Please consider marking this 'fixed' as I think trunk is definitely doing a saner thing these days than it was in your referenced snippet! Thanx.
